### DISCUSSION TITLE
How do I install the bot on linux? [Tutorial]

### DISCUSSION THREAD

**Discussion by d0biar** [Refloow/Steam-Card-Bot-PRO#14] [Q&A] [answered]
How do I install the bot on linux? [Asked by a discord memeber]

**Comment by Refloow** [comment, marked as answer]
### **Firstly install nodejs with the command**

`sudo apt install nodejs`

### **Then install git with the command**

`sudo dnf install git-all`

### **Download (clone) the project repo (files) with the command**

`git clone https://github.com/OSL-Works/Steam-Card-Bot-PRO`

### **Navigate to the projects directory, then install required modules with the command**

`npm install`

### **After installing project's modules configure the config file (more about setting the project up in the wiki):**

**_[Projects WIKI link](https://github.com/OSL-Works/Steam-Card-Bot-PRO/wiki)_**

### **Run the software using the command**

`node index.js`
